xla/service/gpu/gpu_executable.cc

```diff
@@ -80,6 +80,7 @@ limitations under the License.
 #if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
 #include "xla/stream_executor/gpu/gpu_activation.h"
 #include "xla/stream_executor/gpu/gpu_executor.h"
+#include "xla/stream_executor/gpu/gpu_stream.h"
```

Comment:
This include needs a corresponding dependency in the BUILD target.

Reply:
updated